pattern.py

```diff
@@ -84,7 +86,10 @@ def print_help():
     if sys.argv[1].isdigit():
         try:
             pat = pattern_gen(int(sys.argv[1]))
+            print("Pattern copy to Clipboard: \n")
```

Comment:
This reads better if replaced with
```
clipboard.copy(pat)
print("Pattern copied to clipboard: \n")
print(pat)
```
Also remove the newline before the `except`.